kedro/framework/cli/starters.py

```diff
@@ -62,6 +72,34 @@ def _remove_readonly(func: Callable, path: Path, excinfo: Tuple):  # pragma: no
     func(path)
 
 
+def _get_starters_aliases() -> Dict[str, Dict[str, str]]:
+    """This functions lists all the starters aliases declared in
+    the core repo and in plugins entry points.
+    """
+    official_starter_aliases = _STARTER_ALIASES.copy()
+    starters_aliases_from_plugins = load_entry_points("starter")
+
+    # TODO: it would likely be a good idea to indicate in the name
+    # from which plugin this is imported, e.g  we may store the origin
+    # which would return a dict like {pandas-iris: kedro, my_awesome-starter: awesome_plugin}
+    # this is useful for debugging and when listing discovered starters
+
+    # TODO Add the possibility for plugins users to specify {starter_name: git+url}
+    #  instead of {starter_name: {"repo": git+url, "directory": None}}
+    # which is unneedly verbose and error prone
+
+    starters_aliases_from_plugins = {
+        k: v
+        for starter_alias in starters_aliases_from_plugins
+        for k, v in starter_alias.items()
+    }
```

Comment:
What happens if there are multiple plugins that use this entry point? Will they all be there? Will they overwrite each other if `starter_name` is repeated? 🤔

Reply:
I've added a warning  which will print: the first one will be registered, and other will not with a warning message